### PR TITLE
executor, distsql: remove useless code.

### PR DESCRIPTION
**Pull request by hanfei1991**

### CODE REVIEW

File: executor/analyze.go [hunk 1]
```diff
@@ -142,7 +148,6 @@ type analyzeTask struct {
 	indexInfo *model.IndexInfo
 	Columns   []*model.ColumnInfo
 	PKInfo    *model.ColumnInfo
-	src       Executor
```
Comment (lamxTyler) [inline executor/analyze.go]: All the above field can be removed?

File: executor/builder.go [hunk 2]
```diff
@@ -909,35 +831,16 @@ func (b *executorBuilder) buildAnalyze(v *plan.Analyze) Executor {
 		tasks: make([]*analyzeTask, 0, len(v.Children())),
 	}
 	for _, task := range v.ColTasks {
-		if task.PushDown {
```
Comment (lamxTyler) [inline executor/builder.go]: This field can be removed too.

File: executor/analyze.go (hunk 1 above)
Comment (hanfei1991) [reply executor/analyze.go]: These fields are still used.

Comment (lamxTyler) [reply executor/analyze.go]: Where?

File: executor/builder.go (hunk 2 above)
Comment (hanfei1991) [reply executor/builder.go]: removed

File: executor/analyze.go (hunk 1 above)
Comment (hanfei1991) [reply executor/analyze.go]: ok, removed.

**Pull request merged**